addons/website_slides/views/rating_rating_views.xml

```diff
@@ -99,14 +97,72 @@
+        <field name="mode">primary</field>
+        <field name="inherit_id" ref="rating_rating_view_kanban_slide_channel"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[1]/div[1]/div[1]/div[2]" position="replace" />
+            <xpath expr="//div[1]/div[1]/div[1]/div[1]" position="replace" />
+            <xpath expr="//div[1]/div[1]" position="attributes">
+                <attribute name="class">d-flex flex-row</attribute>
+            </xpath>
+            <xpath expr="//kanban" position="attributes">
+                <attribute name="class">o_slide_review_kanban</attribute>
+            </xpath>
+            <xpath expr="//div[1]/div[1]/div[1]" position="attributes">
+                <attribute name="class">o_slide_rating_kanban_left mr-3</attribute>
+            </xpath>
+            <xpath expr="//div[1]/div[1]/div[1]" position="after">
+                <div>
+                    <div class="o_kanban_card_header">
+                        <div class="o_kanban_card_header_title">
+                            <span class="font-weight-bold"><field name="partner_id"/></span>
+                        </div>
+                    </div>
+                    <div class="o_kanban_card_content mt0 d-flex flex-column">
+                        <span>
+                            <i class="fa fa-folder mr-2" aria-label="Open folder"></i>
+                            <a type="object" name="action_open_rated_object" t-att-title="record.res_name.raw_value">
+                                <field name="res_name" />
+                            </a>
+                        </span>
+                        <span><i class="fa fa-clock-o mr-2" aria-label="Create date"/> <field name="create_date" /></span>
+                        <div class="d-flex mt-2">
+                            <span t-esc="record.feedback.raw_value"/>
+                        </div>
+                    </div>
+                </div>
+            </xpath>
+        </field>
+    </record>
+    
     <record id="rating_rating_action_slide_channel" model="ir.actions.act_window">
         <field name="name">Rating</field>
         <field name="res_model">rating.rating</field>
         <field name="view_mode">kanban,tree,graph,pivot,form</field>
-        <field name="domain">[('consumed', '=', True), ('res_model', '=', 'slide.channel')]</field>
+        <field name="domain">[('consumed', '=', True), ('res_model', '=', 'slide.channel'),]</field>
+        <field name="context">{}</field>
+        <field name="search_view_id" ref="rating_rating_view_search_slide_channel"/>
+        <field name="view_id" ref="rating_rating_view_kanban_slide_channel" />
+        <field name="help" type="html">
+            <p class="o_view_nocontent_empty_folder">
+                There are no ratings for these courses at the moment
+            </p>
+        </field>
+    </record>
+
+    <record id="rating_review_action_slide_channel" model="ir.actions.act_window">
```

Comment:
I would gives an id like rating_rating_action_slide_channel_review to keep the "rating_rating" prefix, telling it is an act window on rating.rating model.